FEM/fem_ele.cpp

```diff
@@ -172,6 +175,7 @@ void CElement::ConfigElement(CElem* MElement, const int nquadrature_points, bool
 	{
 		if (dim != ele_dim)
 		{
+			dim_grad = dim;
```

Comment:
No, it can't be moved. It is for inclined elements. The dimensions of local and the global coordinates are different. For example, for a regular line element,  dim_grad = ele_dim = 1, while for a inclined line element in 3 D space,   dim_grad = 3.